.github/workflows/update-gardenlogin.yaml

```diff
@@ -2,63 +2,152 @@ name: gardenlogin-updater
-          --data "${data}"
-      - name: Send update with latest versions to ${{ github.repository_owner }}/chocolatey-packages
-        if: github.event.release.prerelease == false
+          set -euo pipefail
+          data=$(jq -n \
+            --arg component          "gardenlogin" \
+            --arg tag                "$TAG" \
+            --arg darwin_sha_amd64   "$DARWIN_SHA_AMD64" \
+            --arg darwin_sha_arm64   "$DARWIN_SHA_ARM64" \
+            --arg linux_sha_amd64    "$LINUX_SHA_AMD64" \
+            --arg linux_sha_arm64    "$LINUX_SHA_ARM64" \
+            '{event_type:"update",
+              client_payload:{
+                component:$component,
+                tag:$tag,
+                darwin_sha_amd64:$darwin_sha_amd64,
+                darwin_sha_arm64:$darwin_sha_arm64,
+                linux_sha_amd64:$linux_sha_amd64,
+                linux_sha_arm64:$linux_sha_arm64}}')
+          curl -X POST \
+            -H 'Accept: application/vnd.github.everest-preview+json' \
+            -H "Authorization: Bearer ${{ steps.gardener-github-workflows.outputs.token }}" \
+            -d "$data" \
+            "https://api.github.com/repos/${{ github.repository_owner }}/homebrew-tap/dispatches"
+
+      # ───── Chocolatey dispatch ──────────────────────────────────────────────
+      - name: Send update to ${{ github.repository_owner }}/chocolatey-packages
+        if: (github.event_name == 'release' && github.event.release.prerelease == false) || (github.event_name == 'workflow_dispatch' && inputs.dry_run == false)
         run: |
-          latest_release_filtered_tag=${{ env.latest_release_filtered_tag }}
-          windows_sha256sum=$(sha256sum bin/windows-amd64/gardenlogin_windows_amd64.exe | awk '{print $1}')
-          data='{"event_type": "update", "client_payload": { "component": "gardenlogin", "tag": "'"$latest_release_filtered_tag"'", "windows_sha": "'"$windows_sha256sum"'"}}'
-          echo "${data}"
-          curl -X POST https://api.github.com/repos/${{ github.repository_owner }}/chocolatey-packages/dispatches \
-          -H 'Accept: application/vnd.github.everest-preview+json' \
-          -H "Authorization: Token ${{ steps.gardener-github-workflows.outputs.token }}" \
-          --data "${data}"
+          set -euo pipefail
+          data=$(jq -n \
+            --arg component   "gardenlogin" \
+            --arg tag         "$TAG" \
+            --arg windows_sha "$WINDOWS_SHA" \
+            '{event_type:"update",
+              client_payload:{
+                component:$component,
+                tag:$tag,
+                windows_sha:$windows_sha}}')
+          curl -X POST \
+            -H 'Accept: application/vnd.github.everest-preview+json' \
+            -H "Authorization: Bearer ${{ steps.gardener-github-workflows.outputs.token }}" \
+            -d "$data" \
+            "https://api.github.com/repos/${{ github.repository_owner }}/chocolatey-packages/dispatches"
```

Comment:
remove all occurrences of ${{ github.repository_owner }} in the script and introduce an `OWNER` env var instead.